lib/bolt/module_installer.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
-require 'bolt/error'
-require 'bolt/logger'
+require_relative 'error'
```

Comment:
Why the require_relatives?

Reply:
Ah, that made testing the classes in IRB easier. :P Will update.

Reply:
If you `bundle exec irb`, regular `require` statements should work.

Reply:
TIL!